Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM python:3.7-alpine3.9
+FROM python:3.7-stretch
 
-RUN apk add build-base freetype-dev git jpeg-dev libffi-dev zlib-dev
+RUN apt update
+RUN apt install -y build-base freetype-dev git jpeg-dev libffi-dev zlib-dev
```

Comment:
This should be `RUN apt install -y build-essential libfreetype6-dev git libjpeg-dev libffi-dev zlib1g`.

Reply:
Thanks